rootfs/bin/get_object

```diff
@@ -4,8 +4,9 @@ GET_PATH=slug.tgz
 
 export BUCKET_FILE=/var/run/secrets/deis/objectstore/creds/builder-bucket
 if [ "$BUILDER_STORAGE" == "minio" ]; then
+  MINION_BUCKET_NAME=${MINIO_BUCKET:-git}
```

Comment:
Could you make this `MINIO_BUCKET_NAME` (not `MINION`)?

Sorry for the CI failures--Jenkins is having a bad day and I'm babysitting him. I think this is actually passing and I'll make sure it goes green.